src/main/java/io/github/genomicdatainfrastructure/discovery/api/DatasetQueryApiImpl.java

```diff
@@ -17,7 +17,7 @@
 public class DatasetQueryApiImpl implements DatasetQueryApi {
 
     private final SecurityIdentity identity;
-    private final DatasetsSearchService datasetsSearchService;
+    private final BeaconIndividualsSearchService datasetsSearchService;
```

Comment:
maybe call it individualsService to avoid confusion?

Individual means Dataset in Beacon terminology right?

Reply:
Nope, `individuals` means the scope of the data we are querying on. In that case, we are querying on individual (human being) records. Facets in Beacon context are record level variables, like sex, ethnicity, weight, height, diseases...

Reply:
I renamed the classes to `BeaconDatasetsSearchService` and `CkanDatasetsSearchService`, and they are implementing `DatasetsSearchService`, to give better idea of `BeaconDatasetsSearchService` as a decorator of `CkanDatasetsSearchService`, what do you think?

Anyway, there is this https://github.com/GenomicDataInfrastructure/gdi-userportal-dataset-discovery-service/issues/11 to review the current architecture. We are still a bit far from the ideal and we will revise it as soon as we finish the milestone 7.